README.md

```diff
@@ -7,6 +7,13 @@
 
 Orion is a Apache 2.0 licensed, private transaction manager.
 
+## Issues 
+
+Orion issues are tracked in [Jira](https://pegasys1.atlassian.net/secure/Dashboard.jspa?selectPageId=10000) not GitHub. 
+See our [contribution guidelines](CONTRIBUTING.md) for more detail on searching and creating issues. 
+
+If you've commented on existing issue and have been waiting for a reply, feel free to ping us on [Gitter].  
```

Comment:
There were no issues. Should we just say to contact us on Gitter for any questions and keep it generic?

Reply:
Plus Gitter link

Reply:
Gitter link is there in the actual page - it's just at the bottom of the MD file.

Reply:
I did specifically want to call out the case where they've commented on something and there's not been a reply.  I think it's OK?  We say in the Contributing docs to get us on Gitter for any questions.